plugins/upstash/api_key_test.go

```diff
@@ -0,0 +1,59 @@
+package upstash
+
+import (
+	"testing"
+	
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+	
+func TestAPIKeyProvisioner(t *testing.T) {
+	plugintest.TestProvisioner(t, APIKey().DefaultProvisioner, map[string]plugintest.ProvisionCase{
+		"default": {
+			ItemFields: map[sdk.FieldName]string{ 
+				fieldname.APIKey: "d68850db-69f7-rr97jk7wcvshbkuexample",
+				fieldname.Email :  "fakememail12@gmail.com",
+			},
+			ExpectedOutput: sdk.ProvisionOutput{
+				Environment: map[string]string{
+					"UPSTASH_API_KEY": "d68850db-69f7-rr97jk7wcvshbkuexample",
+					"UPSTASH_EMAIL" : "fakememail12@gmail.com",
+				},
+			},
+		},
+	})
+}
+
+func TestAPIKeyImporter(t *testing.T) {
+	plugintest.TestImporter(t, APIKey().Importer, map[string]plugintest.ImportCase{
+		"environment": {
+			Environment: map[string]string{ 
+				"UPSTASH_API_KEY": "d68850db-69f7-rr97jk7wcvshbkuexample",
+				"UPSTASH_EMAIL" : "fakememail12@gmail.com",
+			},
+			ExpectedCandidates: []sdk.ImportCandidate{
+				{
+					Fields: map[sdk.FieldName]string{
+						fieldname.APIKey: "d68850db-69f7-rr97jk7wcvshbkuexample",
+						fieldname.Email :  "fakememail12@gmail.com",
+					},
+				},
+			},
+		},
+		
+		"config file": {
+			Files: map[string]string{
+				"~/.upstash.json": plugintest.LoadFixture(t, "upstash.json"),
+			},
+			ExpectedCandidates: []sdk.ImportCandidate{
+				{
+					Fields: map[sdk.FieldName]string{
+						fieldname.Token: "d68850db-69f7-rr97jk7wcvshbkuexample",
```

Comment:
Your importer uses `fieldname.APIKey`, however, the expected candidate here is `fieldname.Token` - I think this is why the test is failing, let's make sure we use API Key all throughout.

Reply:
@hculea I will do it.

Reply:
@hculea I am a bit unclear on what changed recently because there are some instances of force-pushing to this branch (so the history is lost), but anyway, the fieldname seems to be `APIKey` throughout and not `Token`. The tests seem to pass too, so I'll mark this conversation as resolved.